pulp_rpm/app/tasks/publishing.py

```diff
@@ -156,28 +161,48 @@ def populate(self):
             self.handle_sub_repos(distribution_tree)
 
         all_content = main_content
-        for name, content in self.sub_repos:
+        for name, content, checksums in self.sub_repos:
             os.mkdir(name)
             setattr(self, f"{name}_content", content)
             setattr(self, f"{name}_repomdrecords", self.prepare_metadata_files(content, name))
             all_content |= content
         self.publish_artifacts(all_content)
 
 
-def publish(repository_version_pk, metadata_signing_service=None):
+def get_checksum(cr, name, checksums):
+    """
+    Get checksum algorithm for publishing metadata.
+
+    """
+    original = checksums.get("original")
+    metadata = checksums.get("metadata")
+    package = checksums.get("package")
+    checksum_type = original.get(name, "SHA256")
```

Comment:
Let's not have the checksum type hard-coded in the code. 
Could you add it to other constants? Or maybe we already have it there.